src/Calculator.vue

```diff
@@ -186,36 +33,93 @@
+              type="button"
+              id="dropdownMenuButton"
+              data-toggle="dropdown"
+              aria-haspopup="true"
+              aria-expanded="false"
+            >
+              {{ selectedDropdownItem.title }}
+              <i class="fa fa-chevron-down"></i>
+            </button>
+            <div
+              class="dropdown-menu dropdown-menu-right"
+              aria-labelledby="dropdownMenuButton"
+            >
+              <div
+                class="dropdown-item"
+                v-for="(item, i) in dropdownItems"
+                :key="i"
+                @click="selectedDropdownItem = item"
+              >
+                {{ item.title }}
+              </div>
+            </div>
+          </div>
         </div>
         <div class="right-column__content">
-          <div class="total">Your total payment will be</div>
-          <component
-            :is="calc1"
-            :workload="workload"
-            :reserved="billAnnually"
-            v-model="scyllaPrices"
-          ></component>
-        </div>
-
-
-        <!--initial layout leftovers start-->
-        <div class="row">
-          <div class="col-6 m-x-1">
-            <!--                        <dropdown :options="scyllaCalcs" v-model="calc1" description="Scylla offering"></dropdown>-->
-
-          </div>
-          <div class="col-6 m-x-1 d-none">
-            <dropdown
-              :options="rivalCalcs"
-              v-model="calc2"
-              description="Rival offering"
-            ></dropdown>
+          <div v-if="selectedDropdownItem.name === 'details'">
+            <div class="total">Your total payment will be</div>
             <component
-              :is="calc2"
+              :is="calc1"
               :workload="workload"
-              v-model="rivalPrices"
+              :reserved="billAnnually"
+              v-model="scyllaPrices"
             ></component>
           </div>
+          <div v-else>
+            <div class="row">
+              <div class="col-6 px-0">
+                <!--                        <dropdown :options="scyllaCalcs" v-model="calc1" description="Scylla offering"></dropdown>-->
+                <div class="calculator-header">
+                  <img src="./assets/Scylla.png" :alt="calc1" />
+                  <h3>{{ calc1 }}</h3>
+                </div>
+                <component
+                  :is="calc1"
+                  v-model="scyllaPrices"
+                  :workload="workload"
+                  :reserved="false"
+                  hide-specs
+                ></component>
+                <component
+                  :is="calc1"
+                  v-model="scyllaPrices"
+                  :workload="workload"
+                  :reserved="true"
+                  hide-specs
+                ></component>
+              </div>
+              <div class="col-6 pl-3 pr-0">
+                <div class="calculator-header">
+                  <img
```

Comment:
branching for every possible image is not nice. pass an image name parameter instead, i don't want N img tags with if statements

Reply:
fixed, ran into issue with webpack initially